src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = React.memo((props) => {
```

Comment:
I doubt you need `memo` here, same thing in `TodoList.tsx`

https://react.dev/reference/react/memo#should-you-add-memo-everywhere